### ISSUE TITLE
bar height/width

### ISSUE THREAD

**Issue by hrbrmstr** [TargetProcess/tauCharts#71]
Is there a way to force a certain bar height / width for horizontal/vertical bar charts?

Since I'm using the R htmlwidget version to make this here's an example:

http://rpubs.com/hrbrmstr/taucharts_issue_01

![image](https://cloud.githubusercontent.com/assets/509878/9411582/d0f2191e-47f4-11e5-97ae-7e1cb44d0776.png)

I don't want the bars to vary height/width by category, but I do want them to be taller (wider) than they are now and hacking CSS afterwards means also doing y position modifications and that's doable but it'd be nice to override the auto-calculated hight if possible. I might just be missing something in the doc, tho.

**Comment by vladminsky** [comment]
According to Grammar of graphic it is a "size" parameter for geometry. Currently only stacked bar element supports visual encoding for size.

There are plans to support "size" in "usual" bar element but I can't say you exact time.

You can look at the Taucharts examples page where we build a "funnel" by assigning a measure to a stacked bar size (last example): http://taucharts.com/taulab/index.html

Let me know if you have good thoughts on this feature.. probably some sketches or examples of charts you want to build.

**Comment by hrbrmstr** [comment]
roger that (though I can tweak width in ggplot2 which is prbly the best implementation of GoG to-date :-)

**Comment by vladminsky** [comment]
Looks like I understand you in a wrong way.

Currently it works so since there is an assumption that one slot will contain several or even all "color" categories thus each bar / column is shrinked and placed to its position with some offset.

Let's consider your request as an improvement to be implemented. Currently there is no good workaround for this issue.

**Comment by vladminsky** [comment]
@hrbrmstr,

The fix will be published with the next build (0.7.7). We disable positioning by color for bar chart once color and base axis use the same data property.

Set "enableColorToBarPosition" to enable it back for the case:

```javascript
    "type": "horizontal-bar",
    "x": ["x1"],
    "y": ["y2", "y1"],
    "color": "y2",
    "guide": [
        {}
        ,
        {
            enableColorToBarPosition: true
        }
    ],
    data: [...]
```

Let me know if you have any questions.
